user-docs/pupil-capture.md

```diff
@@ -49,7 +49,8 @@ Pupil uses two cameras. One camera records a subject's eye movements -- we call
 
 > {{< video-webm src="/videos/calibration/calibration-mobo/clb-mobo" >}}
 
-Adjust the headset according to your comfort and to ensure a good calibration result
+Pupil Headset comes in a variety of configuration. Calibration involves two types of configuration, monocular and binocular.
```

Comment:
Edit - pluralize:

configuration > configurations

Edit second sentence - Calibration be conducted with a monocular or binocular eye camera setup.